locales/en/messages.json

```diff
@@ -1183,20 +1183,29 @@
     "initialSetupInfoBuildConfig": {
         "message": "Config"
     },
+    "initialSetupInfoBuildDownload": {
+        "message": "Download"
+    },
+    "initialSetupInfoBuildFirmware": {
+        "message": "Firmware:"
+    },
     "initialSetupInfoBuildLog": {
         "message": "Log"
     },
     "initialSetupInfoBuildOptions": {
         "message": "Options"
     },
+    "initialSetupInfoBuildOptionsLabel": {
```

Comment:
There's no standard I saw for label vs. content, so I appended "Label" here because the plain "Options" string is used elsewhere.